tensorbay/client/job.py

```diff
@@ -233,3 +235,47 @@ def result(self) -> Optional[Draft]:
             return self._draft_getter(draft_number)
 
         return None
+
+    @classmethod
+    def from_response_body(  # type: ignore[override]  # pylint: disable=arguments-differ
+        cls: Type[_T],
+        body: Dict[str, Any],
+        *,
+        client: Client,
+        dataset_id: str,
+        job_updater: Callable[[str], Dict[str, Any]],  # noqa: DAR101
+        draft_getter: Callable[[int], Draft],
+    ) -> _T:
+        """Loads a :class:`SquashAndMergeJob` object from a response body.
+
+        Arguments:
+            body: The response body which contains the information of a SquashAndMergeJob,
+                whose format should be like::
+
+                    {
+                        "title": <str>
+                        "jobId": <str>
+                        "arguments": <object>
+                        "createdAt": <int>
+                        "startedAt": <int>
+                        "finishedAt": <int>
+                        "status": <str>
+                        "errorMessage": <str>
+                        "result": <object>
+                        "description": <str>
+                    }
+            client: The :class:`~tensorbay.client.requests.Client`.
+            dataset_id: Dataset ID.
+            job_updater: The function to update the information of the SquashAndMergeJob instance.
+            draft_getter: The function to get draft by draft_number.
+
+        Returns:
+            The loaded :class:`SquashAndMergeJob` object.
+
+        """
+        job = super().from_response_body(
```

Comment:
the type of `job` is `Job` not `SquashAndMergeJob`??

Reply:
The return type of this function in Job is the type of input cls.